### PR TITLE
Fix width of the password field in mobile view

### PR DESCRIPTION
**Pull request by PriyangaPKini**
**Story card:** [sc-3791](https://app.shortcut.com/simpledotorg/story/3791/password-field-has-set-width-and-goes-beyond-the-edge-of-the-view-on-mobile-set-your-password-create-your-password-views)

## Because

Password field has set width and goes beyond the edge of the view on mobile "Set your password" / "Change your password" views

## This addresses

- Remove the unwanted styling for the password input field
- Fix the position of the `password-toggle-button`
- Style window to match others

## Test instructions
Inspect the window in different dimensions (mobile, tablet, website)

Attaching screenshots:
change-password-after
change-password-website

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -461,8 +461,8 @@ CREATE TABLE public.estimated_populations (
     region_id uuid NOT NULL,
```
Comment (ssrihari) [inline db/structure.sql:461]: Unrelated changes here.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -655,11 +655,6 @@ span.inactive {
 	}
 }
 
-.password-input {
-	height: 44px;
-	width: 480px;
-}
-
```
Comment (ssrihari) [inline app/assets/stylesheets/application.scss:662]: This looks like it was added for a specific reason. How did you ensure that this didn't cause any regression?

**Review by ssrihari — CHANGES_REQUESTED** [review]
One question, and one change needed to remove unrelated changes. Looks good overall, though!

File: db/structure.sql (hunk 1 above)
Comment (PriyangaPKini) [reply db/structure.sql:461]: When I pull the master and ran migration these changes were present. I suppose someone forgot to commit these changes. I had discussed with @prabhanshuguptagit whether to commit these changes here. He asked me to commit them.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (PriyangaPKini) [reply app/assets/stylesheets/application.scss:662]: The `show` button was placed on top of the password input field by setting a particular size for the field. In mobile view, because of the above set width, the password field goes beyond the screen as well as the `show` button. 
After removing these, the size of the field became relative and adjust with the change in dimension of the window.
Fixed the position of the `show` button by specifying `right-margin` instead of `left-margin`

File: db/structure.sql (hunk 1 above)
Comment (ssrihari) [reply db/structure.sql:461]: Can we make this another PR, in that case? It's strange that these are getting missed, and we should deal with it separately.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (ssrihari) [reply app/assets/stylesheets/application.scss:662]: Cool.

File: db/structure.sql (hunk 1 above)
Comment (PriyangaPKini) [reply db/structure.sql:461]: Okay. I will raise a separate PR for this

**Review by claudiovallejo — COMMENTED** [review]
Hi @PriyangaPKini — the screenshots in the PR description look great! I'd love to be able to test this in my local or a test app to test under different browser widths.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -635,16 +635,16 @@ span.inactive {
...
-	line-height: 0;
-	padding: 0px;
-	position: absolute;
-	width: 47px;
-	z-index: 10;
+  color: $blue;
+  bottom: 10px;
+  font-size: 14px;
+  height: 20px;
+  right: 10px;
```
Comment (claudiovallejo) [inline app/assets/stylesheets/application.scss:642]: @PriyangaPKini I tested in my local and have a few minor CSS change suggestions:
1. Make `right: 9px`
2. Remove `width`
3. Set `padding-right: 4px` and `padding-left: 4px`

**Review by claudiovallejo — APPROVED** [review]

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (PriyangaPKini) [reply app/assets/stylesheets/application.scss:642]: Will do these chnages.

**Review by claudiovallejo — APPROVED** [review]
